reference-architectures/deploy-arch-ibm-slz-vsi-quickstart.md

```diff
@@ -96,5 +96,5 @@ business challenge, or target cloud environments.
 
 | Requirement | Component | Reasons for choice | Alternative choice |
 |-------------|-----------|--------------------|--------------------|
-| * Use public and private SSH keys to access virtual server instances by using SSH  \n * Do not store private SSH key on any virtual instances, also not on the jump box host  \n * Do not allow any other SSH login methods except the one with the specified public and private SSH key pair | Public and private SSH keys provided by customer | Ask customer to specify the keys. Accept the input as secure parameter or as reference to the key stored in IBM Cloud Secrets Manager. | |
+| * Use public and private SSH keys to access virtual server instances by using SSH  \n * Do not store private SSH key on any virtual instances, also not on the jump box host  \n * Do not allow any other SSH login methods except the one with the specified public and private SSH key pair | Public and private SSH keys provided by customer | Ask customer to specify the keys. Accept the input as secure parameter or as reference to the key stored in key management. | |
```

Comment:
I'm not sure this statement is correct. There is one required input variable called `ssh_public_key`. I don't think there is any way to "reference to the key stored in key management"? And there is also no input for private SSH keys. Not sure if they are in play here?

Reply:
@ocofaigh I've removed the `Key and password management architecture decisions` table based on this information. It was already removed in the VPC ref arch.